docs/Theming.md

```diff
@@ -441,17 +445,23 @@ const MyAppBar = props => <AppBar {...props} userMenu={MyUserMenu} />;
 
 ### Sidebar Customization
 
-You can specify the `Sidebar` size by setting the `size` property:
+You can specify the `Sidebar` width by setting the `width` and `closedWidth` property on your custom material-ui them:
 
 ```jsx
-import { Sidebar } from 'react-admin';
+import { createMuiTheme } from 'react-admin';
```

Comment:
no need to use the react-admin one here as you're providing the required dimensions.